src/aks-preview/azext_aks_preview/agentpool_decorator.py

```diff
@@ -37,6 +43,9 @@
 PortRange = TypeVar("PortRange")
 IPTag = TypeVar("IPTag")
 
+# constants
+CONST_VIRTUAL_MACHINES = "VirtualMachines"
```

Comment:
Please define the const in `src/aks-preview/azext_aks_preview/_consts.py` and import it from there

Reply:
done